lib/chat_api/conversations/query.ex

```diff
@@ -0,0 +1,43 @@
+defmodule ChatApi.Conversations.Query do
+  import Ecto.Query, warn: false
+
+  alias ChatApi.Conversations.Conversation
+  alias ChatApi.Messages.Message
+
+  def query_by_account(account_id, attrs) do
+    Conversation
+    |> join(
+      :left_lateral,
```

Comment:
Not sure if doing a left lateral here would be the most appropriated way, but I didn't manage to find another way to achieve the `subquery` to retrieve only one message(most recent) by conversations.

Any suggestions?

Reply:
seems reasonable! I'm not aware of any better way to do it :P